packages/gafl-webapp-service/src/pages/terms-and-conditions/terms-and-conditions.njk

```diff
@@ -19,75 +21,150 @@
+                                <ul class='govuk-list govuk-list--bullet'>
+                                    <li>{{ mssgs.licence_conditions_info_bulletpoint_1 }}</li>
+                                    <li>{{ mssgs.licence_conditions_trout_info_bulletpoint_2 }}</li>
+                                    <li>{{ mssgs.licence_conditions_info_bulletpoint_3 }}</li>
+                                    <li>{{ mssgs.licence_conditions_info_bulletpoint_4 }}</li>
+                                    <li>{{ mssgs.licence_conditions_info_bulletpoint_5 }}</li>
+                                    <li>{{ mssgs.licence_conditions_info_bulletpoint_6 }}</li>
+                                </ul>"
                     }
-                  ]
-                }) }}
+                },
+                {
+                    heading: {
+                        text: mssgs.licence_type_radio_salmon
+                    },
+                    content: {
+                        html: "<h2 class='govuk-heading-m'>{{ mssgs.licence_conditions_heading }}</h2>
+                                <p class='govuk-body'>{{ mssgs.licence_conditions_paragraph }}</p>
+                                <ul class='govuk-list govuk-list--bullet'>
+                                    <li>{{ mssgs.licence_conditions_bulletpoint_1 }}</li>
+                                    <li>{{ mssgs.licence_conditions_bulletpoint_2 }}</li>
+                                    <li>{{ mssgs.licence_conditions_salmon_bulletpoint_3 }}</li>
+                                    <li>{{ mssgs.licence_conditions_bulletpoint_4 }}</li>
+                                    <li>{{ mssgs.licence_conditions_bulletpoint_5 }}</li>
+                                    <li>{{ mssgs.licence_conditions_bulletpoint_6 }}</li>
+                                    <li>{{ mssgs.licence_conditions_salmon_bulletpoint_7 }}</li>
+                                </ul>
+                                <h2 class='govuk-heading-m'>{{ mssgs.licence_conditions_important_info }}</h2>
+                                <ul class='govuk-list govuk-list--bullet'>
+                                    <li>{{ mssgs.licence_conditions_info_bulletpoint_1 }}</li>
+                                    <li>{{ mssgs.licence_conditions_salmon_info_bulletpoint_2 }}</li>
+                                    <li>{{ mssgs.licence_conditions_salmon_info_bulletpoint_3 }}</li>
+                                    <li>{{ mssgs.licence_conditions_info_bulletpoint_3 }}</li>
+                                    <li>{{ mssgs.licence_conditions_salmon_info_bulletpoint_5 }}</li>
+                                    <li>{{ mssgs.licence_conditions_info_bulletpoint_4 }}</li>
+                                    <li>{{ mssgs.licence_conditions_info_bulletpoint_5 }}</li>
+                                    <li>{{ mssgs.licence_conditions_info_bulletpoint_6 }}</li>
+                                </ul>"
+                    }
```

Comment:
I thought you were going to put this logic into the getData?

Reply:
work for this will be done in https://eaflood.atlassian.net/browse/IWTF-3088?atlOrigin=eyJpIjoiNzNlMWFmYmZlZjczNDM0OGJkZGIxODJjYmZiZWI1MDAiLCJwIjoiaiJ9